Test/Checkpoint-Migration.Tests.ps1

```diff
@@ -1,105 +1,101 @@
 #Requires -Version 5.1
 Set-StrictMode -Version 'Latest'
 
-& (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-Test.ps1' -Resolve)
+BeforeAll {
+    & (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-Test.ps1' -Resolve)
 
-$checkpointedMigrations = [System.Collections.ArrayList]::new()
-$migrationPaths = [System.Collections.ArrayList]::new()
-$existingSchemaContents = @"
-function Push-Migration
-{
-    Add-Table -Name 'Existing' -Column {
-        int 'ID' -Identity
-    }
-}
-
-function Pop-Migration
-{
-    Remove-Table 'Existing'
-}
-"@
-
-function Init
-{
-    $Global:Error.Clear()
     $script:checkpointedMigrations = [System.Collections.ArrayList]::new()
     $script:migrationPaths = [System.Collections.ArrayList]::new()
-    Start-RivetTest
-}
-
-function GivenMigrationContent
-{
-    param(
-        [Parameter(Mandatory)]
-        [String] $Content,
-
-        [String[]] $Database
-    )
-
-    if(-not $Database )
+    $existingSchemaContents = @"
+    function Push-Migration
     {
-        $Database = $RTDatabaseName
+        Add-Table -Name 'Existing' -Column {
+            int 'ID' -Identity
+        }
     }
 
-    foreach( $databaseName in $Database )
+    function Pop-Migration
     {
-        $path = $Content | New-TestMigration -Name 'CheckpointMigration' -DatabaseName $databaseName
-        $script:migrationPaths.Add($path)
+        Remove-Table 'Existing'
     }
-}
+"@
 
-function Reset
-{
-    param(
-        [String[]] $Database
-    )
+    function Init

```

Comment:
When migrating to Pester 5, there should be a single `Describe` block. Each separate `Describe` block should be removed and only its individual `It` block retained (with a good name that still retains what the test does). `Init` and `Reset` functions should be moved to the `Describe` block and become `BeforeEach` and `AfterEach` blocks.